themes/StarterTheme/config/theme.dist.yml

```diff
@@ -83,6 +83,19 @@ global_settings:
               - category
               - index
 
+  image_types:
+    # When theme will be enabled, all image types will be removed
+    # Template must declare their image type.
```

Comment:
what about multi-store? what if one theme uses some types and other theme different?!